index.html

```diff
@@ -0,0 +1,175 @@
+                <li><a href="#"><img src="./img/desktop/twitter.svg" alt="twitter logo"></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </nav>
+        <section class="logo-company">
+            <img src="./img/logo-mother-coding.svg" alt="logo-company">
+        </section>
+        <div class="mobile-menu-page" id="mobile_menu_page">
+            <div class="effect">
+                <div class="create-x">
+                    <i class="fa-solid fa-x" id="icon-x"></i>
+                </div>
+                <ul class="mobile-links">
+                    <li><a href="about.html" id="about">About</a></li>
+                    <li><a href="#main-program" id="program">Program</a></li>
+                    <li><a href="#future-speakers" id="join">Speakers</a></li>
+                    <li><a href="about.html#future-partners-mobile" id="sponsor">Sponsor</a></li> 
+                </ul>
+            </div>
+        </div>
+   </header>
+   <article class="white-logo-company">
+        <figure class="logo-desktop" id="logo-butterfly">
+                <a href="#"><img src="./img/logo-mother-coding.svg" alt="logo-company-desktop"></a>
+        </figure>
+        <ul>
+            <li class="nav-item"><a href="about.html">About</a></li>
+            <li class="nav-item"><a href="#main-program">Program</a></li>
+            <li class="nav-item"><a href="#future-speakers">Speakers</a></li>
+            <li class="nav-item"><a href="#future-partners">Sponsors</a></li>
+            <li class="button-desktop nav-item" id="campaign"><a href="#">CC Campaign</a></li>         
+        </ul>    
+   </article>
+
+    <section class="hero-container" id="about">
+                        <article class="theme">
+                            <span class="intro-phrase">Welcome beautiful Mothers!!</span>  
+                            <h1 class="hero-title">
+                                 Mother kids Programming Online Conference 
+                            </h1>
+                                
+                        </article>
+                        <article class="text-structure">
+                                <p class="text">For the first time in more than eight years, the International Maternal Newborn Health Conference (IMNHC) 2023 brought together the maternal newborn health communities in Cape Town, South Africa. </p>
+                        </article>
+                        <article class="text-event-date">
+                            <h2 class="event-date">
+                                2023.10.23(FRI)~ 24(SAT)
+                            </h2>
+                            <span class="event-place">
+                                @Museum Cape Town Center and more
+                            </span>
+                        </article>
+            </section>
```

Comment:
[OPTIONAL] this was set to optional since it was missed by the previous reviewer but it is very important to implement these changes as you were asked to replicate the given design 

Kindly increase the fontsize of the first headline and margin and padding to make this section cover 80% of the view height as shown in the design , refer to the images ==>

![Capture d’écran du 2023-08-09 08-59-59](https://github.com/IvonneBenitesRodriguez/capstone-project/assets/59707859/bfd7f516-5438-497d-b9f1-abb1d3660c51)
![Capture d’écran du 2023-08-09 08-59-37](https://github.com/IvonneBenitesRodriguez/capstone-project/assets/59707859/60ce7330-f9da-4116-b4b6-a1cf33b5dc88)

Reply:
Dear @souad988 
Very nice to meet you, 👋🏾
I thank you for the suggestions, 
I modified the issues you suggested me, 
Best regards,